gc/2021/GC-10-05.md

```diff
@@ -0,0 +1,192 @@
+![WebAssembly logo](/images/WebAssembly.png)
+
+## Agenda for the October 5 video call of WebAssembly's Garbage Collection Subgroup
+
+- **Where**: zoom.us
+- **When**: October 5, 4pm-6pm UTC (October 5, 9am-11am Pacific Daylight Time)
+- **Location**: *link on calendar invite*
+
+### Registration
+
+None required if you've attended before. Fill out the form here to sign up if
+it's your first time: https://forms.gle/JehrAB4gWbtHjybt9. The meeting is open
+to CG members only.
+
+## Logistics
+
+The meeting will be on a zoom.us video conference.
+Installation is required, see the calendar invite.
+
+## Agenda items
+
+1. Opening, welcome and roll call
+    1. Opening of the meeting
+    1. Introduction of attendees
+1. Find volunteers for note taking (acting chair to volunteer)
+1. Adoption of the agenda
+1. Proposals and discussions
+    1. Announcement: Proposal update in wider CG
+    1. Discussion: Prototyping and investigation plans (10 - 15 minutes)
+    1. Presentation: dart2wasm (Aske Simon Christenson, 20-30 minutes)
+1. Closure
+
+## Meeting Notes
+
+### Introduction of attendees
+
+- Thomas Lively
+- Aske Simon Christensen
+- Ben Titzer
+- Zhi An Ng
+- Roberto Lublinerman
+- Rick Battagline
+- Ross Tate
+- Zalim Bashorov
+- Lars Hansen
+- Keith Miller
+- Tim Steenvoorden
+- Alon Zakai
+- Jakob Kummerow
+- Andreas Rossberg
+- Conrad Watt
+- Goktug Gokdogan
+- Luke Wagner
+- Francis McCabe
+- Mark (??)
+- Asumu Takikawa
+- Slaval Kuzmich
+
+###  Announcement: Proposal update in wider CG
+
+TL: Let me know if you want to join me, Alon, and Jakob in presenting recent progress on the GC proposal.
+
+### Presentation: dart2wasm (Aske Simon Christenson, 20-30 minutes)
+
+TODO: slides
```

Comment:
@askeksa-google, thanks for presenting! Could you upload or add a link to your slides?